data/Makefile

```diff
@@ -21,11 +21,16 @@ ms:
 	cp msdir/ms_summary_stats ./
 	cp msdir/sample_stats ./
 
+mshot:
+	make -C msHOTdir
+	cp msHOTdir/msHOT ./
+	cp msHOTdir/msHOT_summary_stats ./
```

Comment:
msHot's version of summary_stats is probably identical to ms's, so there's no point in including it. We should probably just remove the code also, no point in keeping the files in our repo.

Reply:
do you mean sample stats? `msHOT_summary_stats` is compiled with the necessary flags to produce stats for verification.

Reply:
yes I do, my bad.